main/templates/main/base.account.html

```diff
@@ -0,0 +1,98 @@
+            {% include "main/snippets/navbar.html" %}
+            <!-- Page container -->
+            <div class="container py-5 mt-4 mt-lg-5 mb-lg-4 my-xl-5">
+                <div class="row pt-sm-2 pt-lg-0">
+                    <!-- Sidebar (offcanvas on screens < 992px) -->
+                    <aside class="col-lg-3 pe-lg-4 pe-xl-5 mt-n3">
+                        <div class="position-lg-sticky top-0">
+                            <div class="d-none d-lg-block" style="padding-top: 80px;"></div>
+                            <div class="offcanvas-lg offcanvas-start" id="sidebarAccount">
+                                <button class="btn-close position-absolute top-0 end-0 mt-3 me-3 d-lg-none"
+                                        type="button"
+                                        data-bs-dismiss="offcanvas"
+                                        data-bs-target="#sidebarAccount"
+                                        aria-label="Close"></button>
+                                <div class="offcanvas-body">
+                                    <div class="pb-2 pb-lg-0 mb-4 mb-lg-5">
+                                        <h3 class="h5 mb-1">{{ request.user.username }}</h3>
+                                        <p class="fs-sm text-body-secondary mb-0">{{ request.user.email }}</p>
+                                    </div>
+                                    {% with request.resolver_match.url_name as url_name %}
+                                        <nav class="nav flex-column pb-2 pb-lg-4 mb-3">
+                                            <h4 class="fs-xs fw-medium text-body-secondary text-uppercase pb-1 mb-2">Account</h4>
+                                            <a class="nav-link fw-semibold py-2 px-0 {% if url_name == 'account-overview' %}active{% endif %}"
+                                               href="{% url 'account-overview' %}">
+                                                <i class="ai-user-check fs-5 opacity-60 me-2"></i>
+                                                Overview
+                                            </a>
+                                            <a class="nav-link fw-semibold py-2 px-0 {% if url_name == 'profile' %}active{% endif %}"
+                                               href="{% url 'profile' %}">
+                                                <i class="ai-settings fs-5 opacity-60 me-2"></i>
+                                                Update Profile
+                                            </a>
+                                            <a class="nav-link fw-semibold py-2 px-0 disabled {% if url_name == 'change-password' %}active{% endif %}"
+                                               href="{% url 'account-overview' %}">
+                                                <i class="ai-wallet fs-5 opacity-60 me-2"></i>
+                                                Change Password
+                                            </a>
```

Comment:
This should be enabled once the `password_change_form.html` and `password_change_done.html` are updated to use this as a base.